lib/rubocop/cop/rspec/empty_example_group.rb

```diff
@@ -35,6 +35,30 @@ module RSpec
       #     pending 'will add tests later'
       #   end
       #
+      # @example configuration
+      # # .rubocop.yml
+      # RSpec:
```

Comment:
I think you should increase indentation. Run `rake` so it builds docs to see for sure